parts/k8s/cloud-init/artifacts/cis.sh

```diff
@@ -34,14 +34,16 @@ assignFilePermissions() {
     blobfuse-flexvol-installer.log
     "
     for FILE in ${FILES}; do
-        touch /var/log/${FILE}
-        chmod 640 ${FILE}
+        DIR=$(dirname "${FILE}")
+        mkdir -p ${DIR} || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
```

Comment:
we need to make intermediate directories that don't exist before touching the file.